deploy/asset-inventory-cloudformation/gomain.go

```diff
@@ -0,0 +1,209 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Do I understand correctly that running this will directly start a new stack?

Is this dev only? That's not how we use on pro, right?

Reply:
Yeah, this is a copy of CSPM devtools. Just like we have `just deploy-cloudformation`, we now get `just deploy-asset-inventory-cloudformation`. Not used in CI, just by developers.